tests/app/notifications/test_validators.py

```diff
@@ -322,3 +324,26 @@ def test_allows_api_calls_with_international_numbers_if_service_does_allow_int_s
     service = create_service(notify_db, notify_db_session, permissions=[SMS_TYPE, INTERNATIONAL_SMS_TYPE])
     result = validate_and_format_recipient('20-12-1234-1234', key_type, service, SMS_TYPE)
     assert result == '201212341234'
+
+
+def test_check_service_email_reply_to_id_where_reply_to_id_is_none():
+    assert check_service_email_reply_to_id(None, None) is None
+
+
+def test_check_service_email_reply_to_id_where_reply_to_id_is_not_found(sample_service, fake_uuid):
+    with pytest.raises(BadRequestError) as e:
+        check_service_email_reply_to_id(sample_service.id, fake_uuid)
+    assert e.value.status_code == 400
+    assert e.value.message == 'reply_to_id does not exist in database'
```

Comment:
The error message should match the name of the form parameter, email_reply_to_id